### PR TITLE
First part of doc updates for testing RMB-793

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -532,8 +540,15 @@ public class InitAPIs implements InitAPI {
 
 ## Adding code to run periodically
 
+This API can be used if your module *instance* needs to perform ongoing tasks.
+Consider using Okapi's timer facility if the task to be performed is "per-tenant"
+and only needs to be executed on one instance and not all instances of the module.
+If the task performs operations on persistent storage, that is typically a sign
+that it should be using the Okapi timer facility (not was is presented in this section).
```
Comment (julianladisch) [inline README.md:547]:
```suggestion
that it should be using the Okapi timer facility (not what is presented in this section).
```

Comment (julianladisch) [inline README.md:547]: Can you add a link to https://github.com/folio-org/okapi/blob/master/doc/guide.md#timer-interface ?

File: README.md [hunk 2]
```diff
@@ -348,13 +348,21 @@ Adjust the POM file to match your project, e.g. artifactID, version, etc.
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>domain-models-runtime</artifactId>
-      <version>29.0.0</version>
+      <version>32.3.0</version>
```
Comment (hjiebsco) [inline README.md:351]: `32.3.0` or `32.2.0`?

File: README.md [hunk 3]
```diff
@@ -348,13 +348,21 @@ Adjust the POM file to match your project, e.g. artifactID, version, etc.
...
+    </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>postgres-testing</artifactId>
+      <version>32.3.0</version>
```
Comment (hjiebsco) [inline README.md:356]: `32.3.0` or `32.2.0`?

**Review by hjiebsco — APPROVED** [review]
Did you mean `32.3.0` or `32.2.0`? Both RMB-793 (the doc) and RMB-483 (the work) are marked as fix in `32.2.0`.

File: README.md (hunk 2 above)
Comment (adamdickmeiss) [reply README.md:351]: > Did you mean `32.3.0` or `32.2.0`? Both RMB-793 (the doc) and RMB-483 (the work) are marked as fix in `32.2.0`.

Yep. My bad. Will fix that.

File: README.md (hunk 1 above)
Comment (adamdickmeiss) [reply README.md:547]: sure

Comment (adamdickmeiss) [reply README.md:547]: thx

File: README.md (hunk 3 above)
Comment (adamdickmeiss) [reply README.md:356]: yep

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

**Pull request merged**